packages/web-wallet/src/constants.ts

```diff
@@ -0,0 +1,66 @@
+// Import wallet-lib constants
+import { constants as walletLibConstants } from '@hathor/wallet-lib';
+
+// Network configuration
+export const NETWORKS = {
+  MAINNET: 'mainnet',
+  TESTNET: 'testnet'
+} as const;
+
+// WafpWYepbV13FVM9Qp9brmBTXgjrn3dnfx 
+// Hathor API endpoints
+export const HATHOR_API_URLS = {
+  MAINNET: 'https://node.hathor.network/v1a',
+  TESTNET: 'https://node1.india.testnet.hathor.network/v1a'
+} as const;
+
+// Hathor explorer URLs
+export const HATHOR_EXPLORER_URLS = {
+  MAINNET: 'https://explorer.hathor.network',
+  TESTNET: 'https://explorer.testnet.hathor.network'
```

Comment:
This explorer will not connect to the same network as the testnet node

Reply:
I'm sorry, at some point I saw that testnet explorer was still connected to testnet-golf and not india, this has since been fixed.